scene/gui/scroll_bar.cpp

```diff
@@ -654,6 +664,11 @@ void ScrollBar::_bind_methods() {
 	BIND_THEME_ITEM_CUSTOM(Theme::DATA_TYPE_ICON, ScrollBar, decrement_icon, "decrement");
 	BIND_THEME_ITEM_CUSTOM(Theme::DATA_TYPE_ICON, ScrollBar, decrement_hl_icon, "decrement_highlight");
 	BIND_THEME_ITEM_CUSTOM(Theme::DATA_TYPE_ICON, ScrollBar, decrement_pressed_icon, "decrement_pressed");
+
+	BIND_THEME_ITEM_CUSTOM(Theme::DATA_TYPE_CONSTANT, ScrollBar, padding_left, "padding_left");
```

Comment:
CUSTOM is for when item name differs from internal name.
```suggestion
	BIND_THEME_ITEM(Theme::DATA_TYPE_CONSTANT, ScrollBar, padding_left);
```